templates/20-04/aws.html

```diff
@@ -0,0 +1,232 @@
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-dxdx74ozlxsl2">Launch a new instance of Ubuntu 22.04 LTS</a>
+              </li>
+            </ul>
+            <h4 class="p-heading--5">For Ubuntu 24.04 LTS (coverage through 2029):</h4>
+            <hr class="p-rule--muted u-no-margin--bottom" />
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet"><a href="/server/docs/upgrade-introduction">Upgrade to Ubuntu 22.04 LTS first and then upgrade to 24.04 LTS</a></li>
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-kbkyvwyqubv2g">Launch a new instance of Ubuntu 24.04 LTS</a>
+              </li>
+              <li class="p-list__item has-bullet">
+                <a href="#get-in-touch">Contact us to purchase annual subscription tokens</a>
+              </li>
+            </ul>
+          </div>
+        </li>
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">
+            Attach an Ubuntu Pro 20.04 subscription to expand your security maintenance (support through 2030).
+          </h3>
+          <div class="p-stepped-list__content">
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet">
+                <a href="/tutorials/how-to-upgrade-ubuntu-lts-to-ubuntu-pro-on-aws-using-aws-license-manager#1-overview">In-place upgrade from Ubuntu 20.04 to Ubuntu Pro 20.04 with no downtime</a>
+              </li>
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-ngmnrama2krsg">Fresh installation of Ubuntu Pro 20.04</a>
+              </li>
+              <li class="p-list__item has-bullet">
+                <a href="#get-in-touch">Contact us to purchase annual subscription tokens</a>
+              </li>
+            </ul>
+
+            <hr class="p-rule--muted" />
+            <h4 class="p-heading--5">Ubuntu Pro 20.04 LTS expanded security maintenance (ESM) for your entire software supply chain, includes:</h4>
+            <ul class="p-list--divided u-no-margin--bottom">
+              <li class="p-list__item has-bullet">
+                The Linux kernel, Ubuntu base OS
+              </li>
+              <li class="p-list__item has-bullet">
+                Key infrastructure components
+              </li>
+              <li class="p-list__item has-bullet">
+                Over 25,000 open-source applications until 2030, adding 5 years of support. It includes
+              </li>
+              <li class="p-list__item has-bullet">
+                24-hour critical vulnerability fixes, Landscape, kernel livepatching, hardening scripts based on CIS and DISA-STIG standards, and FIPS 140 compliance, ensuring your infrastructure meets U.S. government security requirements for cryptographic modules.
```

Comment:
```suggestion
                24-hour critical vulnerability fixes, Landscape, kernel livepatching, hardening scripts based on CIS and DISA-STIG standards, and FIPS 140 compliance, ensuring your infrastructure meets U.S. government security requirements for cryptographic modules
```
Left a comment in figma as well